python/chronos/src/bigdl/chronos/data/tsdataset.py

```diff
@@ -1191,3 +1191,58 @@ def get_cycle_length(self, aggregate='mode', top_k=3):
             self.best_cycle_length = int(res.max())
 
         return self.best_cycle_length
+
+    def export_jit(self, drop_dt_col=True):
+        """
+        Exporting data processing pipeline to torchscript so that it can be used without
+        Python environment. For example, when you are deploying a trained model in C++
+        and need to process input data, you can call this method to get a torchscript module
+        containing the data processing pipeline and save it in a .pt file when you finish
+        developing the model, when deploying, you can load the torchscript module from .pt
+        file and run the data processing pipeline in C++ using libtorch APIs, and the output
+        tensor can be fed into the trained model for inference.
+
+        Currently only scale and roll are supported, the pipeline is similar to the following code:
+        >>> tsdata.scale(scaler, fit=False)\
+        >>>       .roll(lookback, horizon, is_predict=True)
+        The output tensor has same shape and value as tsdata.to_numpy().
+
+        Currently there are some limitations:
+        1. Please make sure the value of each column can be converted to Pytorch tensor,
+        for example, id "00" is not allowed because str can not be converted to a tensor,
+        you should use integer (0, 1, ..) as id instead of string.
+        2. Some features in tsdataset.scale and tsdataset.roll are unavailable in this pipeline:
+            a. If self.roll_additional_feature is not None, it can't be processed in scale and roll
+            b. id_sensitive, time_enc and label_len parameter is not supported in roll
```

Comment:
3. Users are expected to call `.scale(scaler, fit=True)` before calling `export_jit`. Single `roll` operation is not supported for converting now.